src/oneapi/oneapi.pyx

```diff
@@ -24,7 +24,7 @@ from libcpp.string cimport string as std_string
 
 cdef extern from "oneapi/oneapi.h":
     cdef cppclass PySyclExecutionContext:
-        PySyclExecutionContext(const std_string & dev) except +
+        PySyclExecutionContext(const std_string & dev, const bool from_python) except +
```

Comment:
I think, there is no need for such updates. Adding warnings here https://github.com/intel/scikit-learn-intelex/blob/164708ac2160c722a4eb11f233f7a44a84163c37/src/oneapi/oneapi.pyx#L114:L119 would be enough in my opinion.

Reply:
I vote not to change this function signature. Its already deprecated, we can use it as is.

Reply:
last comment can be ignored. since this pr follow changes from https://github.com/oneapi-src/oneDAL/pull/2480